# You can find control table of Dynamixel on emanual (http://emanual.robotis.com/#control-table)
# Control table item has to be set Camel_Case and not included whitespace
# You are supposed to set at least Dynamixel ID
# This example is compatible with Dynamixel using protocol 2.0
servo0:
  ID: 1
  Return_Delay_Time: 0
  Operating_Mode: 1
  Profile_Acceleration: 0
  Profile_Velocity: 0
servo1:
  ID: 2
  Return_Delay_Time: 0
  Operating_Mode: 1
  #Drive_Mode: 4
  Profile_Acceleration: 0
  Profile_Velocity: 0
# servo2:
#   ID: 3
#   Return_Delay_Time: 0
#   Operating_Mode: 1
#   #Drive_Mode: 4
#   Profile_Acceleration: 0
#   Profile_Velocity: 0
# servo3:
#   ID: 4
#   Return_Delay_Time: 0
#   Operating_Mode: 1
#   #Drive_Mode: 4
#   Profile_Acceleration: 0
#   Profile_Velocity: 0
